scripts/check-coding-style.sh

```diff
@@ -1,9 +1,15 @@
 #!/bin/bash
 
-script_dir=$(dirname "$(readlink -f "$0")")
-
+READLINK=$(which greadlink)
+[ -z "$READLINK" ] && {
+	READLINK=$(which readlink)
+}
+[ -z "$1" ] && {
+	echo "USAGE: $0 FILE"
+	exit 1
+}
+CHECKSOURCE=$($READLINK -f "$1")
```

Comment:
My idea of the script was, that it checks automatically all files that should be checked and can be 1:1 run from travis.
So you don't need knowledge about which files were already converted.
With the change it now takes the files to be checked as commandline parameter?